### PR TITLE
Create no_no.json

### PR DESCRIPTION
**Pull request by Garluss**
English translated to Norsk Bokmål
- Garluss

### CODE REVIEW

File: src/main/resources/assets/robotics/lang/no_no.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+  "entity.robotics.clockcopter": "Klokkekopter",
+  "entity.robotics.code_drone": "Kodedrone",
+  "entity.robotics.extend_o_boots": "Ekstend O Støvler",
+  "entity.robotics.miner": "Gruvearbeider",
+
+  "item.robotics.bronze_ingot": "Bronsebarre",
+  "item.robotics.bronze_nugget": "Bronseklump",
+  "item.robotics.clockcopter": "Klokkekopter",
+  "item.robotics.code_drone": "Kodedrone",
```
Comment (OutCraft-Mods) [inline src/main/resources/assets/robotics/lang/no_no.json]: The Name of the Code Drone is Juan, I'd suggest leaving it as-is:
```suggestion
  "item.robotics.code_drone": "Juan",
```

File: src/main/resources/assets/robotics/lang/no_no.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
+  "_": "->------------------------]  Spill Elementer  [------------------------<-",
```
Comment (OutCraft-Mods) [inline src/main/resources/assets/robotics/lang/no_no.json:2]: Nothing big, there's a `{` missing here:
```suggestion
{
  "_": "->------------------------]  Spill Elementer  [------------------------<-",
```

**Review by OutCraft-Mods — CHANGES_REQUESTED** [review]
Hi, everything great, I just have two small suggestions.
Could you look into them?

**Comment by Garluss** [comment]
I will change it when I have time

søn. 25. jun. 2023, 20:47 skrev OutCraft ***@***.***>:

**Review by OutCraft-Mods — APPROVED** [review]

**Comment by OutCraft-Mods** [comment]
Thank you for translating the Mod and fixing the small issues!

**Pull request merged**